web/src/main/java/com/epam/esm/web/controller/GiftCertificateController.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.esm.web.controller;
+
+import com.epam.esm.service.dto.GiftCertificateTagDto;
+import com.epam.esm.service.dto.GiftCertificatesNoTagDto;
+import com.epam.esm.service.dto.SpecificationDto;
+import com.epam.esm.service.exception.ValidationException;
+import com.epam.esm.service.service.GiftCertificateService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/gifts")
```

Comment:
This is about /gifts or /giftCertificates?

Reply:
fixed